config/lobby/db/002_update_tables

```diff
@@ -0,0 +1,25 @@
+alter table ta_users alter column email type varchar(254);
```

Comment:
##### File name

Is there a short phrase that describes the changes encapsulated in this PR?  "Update tables" is kind of generic.  If we stuck with that, I could see every script from now on being called `003_update_tables`, `004_update_tables`, etc. :smile:

For example, if this PR was focused on just increasing the `email` column length, it could be `002_increase_email_length`.  Obviously, there's a lot of changes going on in this PR, so I don't really have any good suggestions right now.  Referencing the PR number is a possibility, but that seems like punting.  However, it has the advantage that a large number of changes can be captured in a short bit of text, although it forces the dev to have to lookup the PR to know what the gist of the changes are.

##### Transaction

All of this code should be run within a transaction.

The first line in the file should be [`BEGIN;`](https://www.postgresql.org/docs/9.5/static/sql-begin.html) or [`START TRANSACTION;`](https://www.postgresql.org/docs/9.5/static/sql-start-transaction.html) I'm not sure which is idiomatic Postgres, but it looks like `BEGIN` is a Postgres extension, while `START TRANSACTION` is standard SQL, so the latter may be preferable.

The last line in the file should be [`COMMIT;`](https://www.postgresql.org/docs/9.5/static/sql-commit.html).

Also, we should ensure that any error in between the first and last lines triggers psql to stop running further commands so we get an implicit `ROLLBACK`.  I'm not familiar with how this is done in Postgres, but in Oracle and MS-SQL, there are global variables you can check for the "error status" of the previously executed command.  I found [this post](http://blog.nordeus.com/dev-ops/psql-exit-on-first-error.htm) which may be a good starting point for us (setting a switch on the command line seems to be preferable to adding psql commands to the script).  You will probably have to force an error during your testing to ensure the script exits correctly and leaves the database unchanged.  Whatever you do here will most likely become our template for error handling in future scripts.

Reply:
@ssoloff Keeping this filename, as this PR does a lot of "smaller improvements" and I really don't want to fiddle around with the git history, changed the filename o n the password PR though as it was just redoing the latest commit and made a lot more sense there.

Reply:
@ssoloff I believe the STOP_ON_ERROR setting should be applied in the travis file and the server config itself, not in the script.

Reply:
@RoiEXLab Agreed.  But we should make sure that it has the desired effect during testing instead of waiting to find out it doesn't work as we expected in production. 😄